docs/components/Form/Mobile.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Content, Form, InputText } from "@jobber/components-native";
+
+export default {
+  title: "Components/Forms and Inputs/Form/Mobile",
+  component: Form,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
```

Comment:
I'm thinking moving forward for all mobile examples we should add `viewport: { defaultViewport: "mobile1" },` within the parameters so that it's clear this is a mobile example.